packages/playground/src/index.html

```diff
@@ -40,6 +40,10 @@
       constants = ethers.constants;
       providers = ethers.providers;
       database = firebase.database;
+      // nullify these as they're expected as globals for the node
+      // but aren't used browserside
+      pg = null;
```

Comment:
Might not need this, or it might not be the same @cf19drofxots.

Reply:
This is a workaround that we unfortunately still need. It's a small thing so I wouldn't worry about it for now.